fpdf/__init__.py

```diff
@@ -39,6 +41,8 @@
     "__license__",
     # Classes
     "FPDF",
+    "X",
```

Comment:
This is really short for a class name...
It's not good if you read `from fpdf import X, Y` and you really don't know what are those `X` & `Y` objects!
I suggest to rename those enums `XAlign` & `YAlign`.
What do you think @gmischler?

Reply:
I see your point. They're not really alignments, though...
`XPos` & `YPos` ?

Reply:
OK, I'm fine with that!